addons/website_quote/controllers/main.py

```diff
@@ -176,15 +160,15 @@ def add(self, option_id, order_id, token, **post):
             'website_description': option.website_description,
             'name': option.name,
             'order_id': order.id,
-            'product_id' : option.product_id.id,
+            'product_id': option.product_id.id,
             'product_uos_qty': option.quantity,
             'product_uos': option.uom_id.id,
             'product_uom_qty': option.quantity,
             'product_uom': option.uom_id.id,
             'discount': option.discount,
         })
-        line = request.registry.get('sale.order.line').create(request.cr, SUPERUSER_ID, vals, context=request.context)
-        option_obj.write(request.cr, SUPERUSER_ID, [option.id], {'line_id': line}, context=request.context)
+        order_line = request.env['sale.order.line'].sudo().create(vals)
+        option.write({'line_id': order_line.id})
```

Comment:
Currently while rendering template, scenario used is like if sale_order_option_line is not going to display if it has line_id(sale_order_line).so while adding suggested product(sale_order_option) to sale order line, sale order line id is write to sale order option.